main/recom.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pandas as pd
+from lockfile import LockFile
+from time import sleep
+
+
+def start():
+    if not os.path.exists('recom_data.csv'):
+        with open('recom_data.csv', 'w') as fp:
+            fp.write("Files")
+
+
+lock = LockFile('recom_data.csv')
+
+
+def get(user_id):
+    user_id = str(user_id)
+    df = pd.read_csv('recom_data.csv', index_col='Files')
+    
+    if user_id in df.columns:
+        if len(df)>0:
+            corr = df.corr()[user_id]
+            wght = df*corr
+            wght = wght.loc[:, wght.columns != user_id]
+
+            wght['mean'] = wght.mean(axis=1)
+
+            final = wght[['mean']]
+            final = final.sort_values(by=['mean'], ascending=False)[:10]
+
+            return final.index.tolist()   
+        
+        else:

```

Comment:
If the possibilities that come out from the `if` statement are mutually exclusive, then you don't need the `else` statement. For example,
```
if 2 > 0:
    return 2
else:
    return 0
```
can be written as:
```
if 2 > 0:
    return 2

return 0
```
In your code, you could write:
```
if user_id in df.columns:
    if len(df)>0:
        corr = df.corr()[user_id]
        ...
        return final.index.tolist()   

    return []


while lock.is_locked():
    sleep(0.05)

...

return []
```

This applies to a lot of other places in your code. But do check the logic to ensure that mutual exclusivity is present.